### DISCUSSION TITLE
Missing Content

### DISCUSSION THREAD

**Discussion by Deathe25** [maxiride/pf-encounters#32] [Q&A]
Hey i love your Builder i use it all the time for my games but i just recently noticed that you missed the stat blocks from the game mastery guide an wanted to know if you plane to add them to the builder any time

**Comment by maxiride** [comment]
I will have to check if the data is available in a programmatic way or a hand input is needed.

**Comment by apoapostolov** [comment]
@maxiride you can try importing from pf2easy.com

**Comment by Deathe25** [comment]
i noticed when i was tryin to balance a fight an noticed that there were none of the stat blocks for the npcs in gamemastery guide i feel bad on makin you guys work harder but i just wanted to point it out if it wasnt brought up before

**Comment by maxiride** [comment]
New data is available with the downloadable json from AON. It will be integrated in the next release.
